s3/test.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM cgr.dev/chainguard/go as build
+
+WORKDIR /src/s3
+COPY --from=common . ../libs/
+COPY . .
+RUN --mount=type=cache,target=/root/go/pkg \
+    --mount=type=cache,target=/root/.cache/go-build \
+    ls ../ &&\
+    go build -o s3 .
+
+#FROM ghcr.io/acorn-io/aws/utils/cdk-runner:v0.6.0 as cdk-runner
+FROM cgr.dev/chainguard/wolfi-base
+RUN apk add -U --no-cache nodejs bash busybox jq curl zip && \
+    apk del --no-cache wolfi-base apk-tools
+RUN npm install -g aws-cdk
+WORKDIR /app
+COPY ./cdk.json ./
+COPY ./scripts ./scripts
+COPY --from=utils ./scripts/ ./scripts/
+COPY --from=utils ./cdk-runner/cdk-runner .
+COPY --from=utils ./testman/differ ./hooks/pre-change-set-apply
```

Comment:
I think overriding this might cause an issue. The RDS dbs all have this hook, and it would be nice to test that the hook itself is doing the right thing.

Do we need it's own hook?

Reply:
Split into it's own `dry-run` hook now